pkgs/development/python-modules/pymodes/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchPypi, buildPythonPackage, numpy, pyzmq }:
+
+buildPythonPackage rec {
+  pname = "pyModeS";
```

```suggestion
  pname = "pymodes";
```

Naming should follow [PEP-503](https://nixos.org/manual/nixpkgs/stable/#contributing).

---

Fixed.